deploy/examples/object-openshift.yaml

```
@@ -119,6 +119,9 @@ spec:
   #        VAULT_BACKEND: v2
   #     # name of the secret containing the kms authentication token
   #     tokenSecretName: rook-vault-token
+  #   serverSideEncryption: # type of server side encryption used "kms", "s3" or both
+  #     kmsEnabled: true
+  #     # s3Enabled: true
```

Why double comment it? What about this?
```suggestion
  #     s3Enabled: false
```